main/acle.md

```diff
@@ -9382,6 +9415,2348 @@ are named after. All of the functions have external linkage.
+                                uint64_t slice_x2_offset, svint8x2_t zn);
+  
+  
+  // Variants are also available for _za8[_u8], _za16[_s16], _za16[_u16],
+  // _za16[_f16], _za16[_bf16], _za32[_s32], _za32[_u32], _za32[_f32],
+  // _za64[_s64], _za64[_u64] and _za64_[f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_hor_za8[_s8]_vg4(uint64_t tile, uint32_t slice_base,
+                                uint64_t slice_x4_offset, svint8x4_t zn);
+  
+  
+  // Variants are also available for _za8[_u8], _za16[_s16], _za16[_u16],
+  // _za16[_f16], _za16[_bf16], _za32[_s32], _za32[_u32], _za32[_f32],
+  // _za64[_s64], _za64[_u64] and _za64_[f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_ver_za8[_s8]_vg2(uint64_t tile, uint32_t slice_base,
+                                uint64_t slice_x2_offset, svint8x2_t zn);
+  
+  
+  // Variants are also available for _za8[_u8], _za16[_s16], _za16[_u16],
+  // _za16[_f16], _za16[_bf16], _za32[_s32], _za32[_u32], _za32[_f32],
+  // _za64[_s64], _za64[_u64] and _za64_[f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_ver_za8[_s8]_vg4(uint64_t tile, uint32_t slice_base,
+                                uint64_t slice_x4_offset, svint8x4_t zn);
+  
+  
+  // Variants are also available for _za64[_u64] and _za64[_f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_za64[_s64]_vg1x2(uint32_t slice_base, uint64_t slice_offset,
+                                svint64x2_t zn);
+  
+  
+  // Variants are also available for _za64[_u64] and _za64[_f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_za64[_s64]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                svint64x4_t zn);
+  
+  ```
+
+#### PTRUE
+
+Initialise predicate-as-counter to all active
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svptrue_c8();
+  
+  ```
+
+#### PEXT
+
+Transform a predicate-as-counter to a predicate (pair).
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svbool_t svpext_lane_c8(svcount_t pnn, uint64_t imm);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svboolx2_t svpext_lane_c8_x2(svcount_t pnn, uint64_t imm);
+  
+  ```
+
+#### PSEL
+
+Predicate select between predicate value or all-false
+
+``` c
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svbool_t svpsel_lane_b8(svbool_t pn, svbool_t pm, uint32_t idx, uint64_t imm);
+  
+  ```
+
+#### CNTP
+
+Set scalar to count from predicate-as-counter. ``vl`` is expected to be 2 or 4.
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  uint64_t svcntp_c8(svcount_t pnn, uint64_t vl);
+  
+  ```
+
+#### UCLAMP, SCLAMP, FCLAMP
```

Comment:
SCLAMP and UCLAMP  single vector are SME
FCLAMP single vector is SME2
https://developer.arm.com/documentation/ddi0602/2022-09/SVE-Instructions/SCLAMP--Signed-clamp-to-minimum-maximum-vector-?lang=en
https://developer.arm.com/documentation/ddi0602/2022-09/SVE-Instructions/UCLAMP--Unsigned-clamp-to-minimum-maximum-vector-?lang=en
https://developer.arm.com/documentation/ddi0602/2022-09/SVE-Instructions/FCLAMP--Floating-point-clamp-to-minimum-maximum-number-?lang=en

Reply:
Sorry, I thought I already answered this question. 

The *multi-vector* UCLAMP/SCLAMP described in this section are SME2.

https://developer.arm.com/documentation/ddi0602/2022-09/SME-Instructions/UCLAMP--Multi-vector-unsigned-clamp-to-minimum-maximum-vector-?lang=en
https://developer.arm.com/documentation/ddi0602/2022-09/SME-Instructions/SCLAMP--Multi-vector-signed-clamp-to-minimum-maximum-vector-?lang=en